docs/BR.md

```diff
@@ -2858,7 +2876,18 @@ This section contains several fields that are common among multiple certificate
 
 ##### 7.1.2.11.2 CRL Distribution Points
 
-If present, the CRL Distribution Points extension MUST be formatted as follows:
+The CRL Distribution Points extension MUST be present in:
+- Subordinate CA Certificates; and 
+- Subscriber Certificates with validity greater than or equal to ten days.
```

Comment:
In #402, @BenWilson-Mozilla commented:

```Change to 5 days, 432,000 seconds?```

Open for discussion (described in open comment above). This value will be set based on the validity of a short-lived certificate.

Reply:
Closing stale comments.